protocol/contracts/amo/Ramos.sol

```diff
@@ -267,7 +267,7 @@ contract RAMOS is Ownable, Pausable {
     function depositStable(
```

Comment:
^^^ Commented here for the rebalanceDown() function.

I spotted a typo previously that we should fix while we have the opportunity, in the RebalanceDown event.

bptIn => bptOut
```
event RebalanceDown(uint256 templeAmountIn, uint256 bptOut);
```